apps/zero-runtime-next-app/src/app/theme.ts

```diff
@@ -0,0 +1,13 @@
+// theme for MUI System (emotion)
```

Comment:
A lot of demos are using `Stack` component by targeting `margin` of its children. The converted component styles are extracted so margin can not be overridden by Stack. We can't change the demos, so need to use ThemeProvider to change to gap instead.